slang-gfx.h

```diff
@@ -1268,6 +1255,34 @@ class ICommandQueue : public ISlangUnknown
         0x14e2bed0, 0xad0, 0x4dc8, { 0xb3, 0x41, 0x6, 0x3f, 0xe7, 0x2d, 0xbf, 0xe } \
     }
 
+class ITransientResourceHeap : public ISlangUnknown
+{
+public:
+    struct Desc
+    {
+        size_t constantBufferSize;
```

Comment:
Comment? It isn't clear what precisely this represents. I'd have to guess it is a hint on the amount of constant buffer memory to pre-allocate for use on ring-buffered CB allocations.

Ideally I'd like it if all the values in the `Desc` here were hints and not requirements, so that performance-conscious apps could set a value they know will be sufficient to avoid runtime allocations, while simple applications can just leave things zeroed at the cost of suffering some on-the-fly allocations during the first few frames that use a given heap.